catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/NsiliEndpoint.java

```diff
@@ -249,13 +249,13 @@ public void init() {
         try {
             initCorba();
         } catch (InvalidName | AdapterInactive | WrongPolicy | ServantNotActive e) {
-            LOGGER.warn("Unable to start the CORBA server. {}",
+            LOGGER.info("Unable to start the CORBA server. Set to DEBUG for a full stack trace {}",
```

Comment:
I was curious about how we felt about the double logging and telling the admin to set the log level to see more. This should probably be mentioned somewhere in the log levels wiki.